care/facility/tasks/redis_index.py

```diff
@@ -15,6 +16,13 @@
 
 @shared_task
 def load_redis_index():
+    try:
+        deleted_count = BaseCommand.delete_pattern("care_static_data*")
+        logger.info("Deleted %s keys with prefix 'care_static_data'", deleted_count)
+    except Exception as e:
+        logger.info("Failed to delete keys with prefix 'care_static_data': %s", e)
```

Comment:
Can we log this as an exception itself ? info logs might not be picked up by sentry

Reply:
Thanks for the tip. I've changed line 23 to say ``logger.exception("Failed to delete keys with prefix 'care_static_data': %s", e)`` instead. 

On a side note, how come the logger commands on line 47 and 49 say ``logger.info()`` when they're treated as exceptions?
![image](https://github.com/user-attachments/assets/15cdccce-6eb6-497e-8ff4-e79ba07aa460)

Reply:
yeah, it would be good to update the log statement on line 49 to be logger.exception but, the one on line 47 is just for debugging so logger.debug would be used for that, you can include these changes in your pr